website/docs/integrator-section/verifiable-document/did/oa-embedded-html.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Embed OA Doc in HTML File
+sidebar_label: Embed OA Doc in HTML File
+---
+
+This step is identical to the one [in the Ethereum tutorial](/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html).
```

Comment:
Do we want to mention that this HTML embedding is optional? It might be a bit confusing because it is technically not a "step" in terms of OA but rather an additional feature

Reply:
Updated the page in Ethereum and DID flow to state that the guide is optional - https://github.com/Open-Attestation/documentation-website/pull/164/commits/57225d18dbf24ba36e016dfae82d0af490d3c658